docs/rfcs/RFC0002-MonoRepo-Gardener-Rotations.md

```diff
@@ -0,0 +1,137 @@
+that other developers do not such as ability to bypass any checks in CI.
+
+Note this concept isn't novel and can be referred to as build cops or build sheriffs
+in other projects. Good judgement and commitment are critical to being a good gardener.
+
+## Overview
+
+### Responsibilities and priorities
+
+1. When regressions occur that break CI workflows, revert those changes.
+If it's not obvious how to, facilitate getting changes in to get back to green (see 2)
+2. For changes that need to bypass presubmit checks (to get back to green), serve as a
+central point of contact to help land those changes using your judgement.
+3. File issues for (or add) presubmit tests that could have caught regressions that were fixed
+in 1/2.
+4. Encourage and contribute to best developer practices
+5. Engage with the infrastructure team for anything that you don't know (they are 
+also part of the gardener chat)
+6. Keep notes for the next gardener in a shared log or communicate via our gardener chat
+
+The gardener is expected to be the primary contact while "on-duty" for
+any changes that break workflows, during their business hours. We will endeavor to have
+gardeners in every time zone that matters so that we can have one available whenever
+they are needed.
+
+This is _not_ an "on-call" position and responses outside of working hours are
+not expected. In the event that a failure occurs while there is no gardener available,
+a list of non on-duty gardeners will be available who have shared superpowers and similar duties.
+
+> [!NOTE]
+> The gardener does not need to personally fix every issue/incident
+> themselves, but they should be proactively monitoring project health and
+> reactively responding to reports as they come in. When responding to an issue,
+> the gardener may delegate to local project developers that follow up
+> to ensure that PRs get reverted.
+
+### Prerequisites for gardening
+
+You will need to be part of the corresponding GitHub gardener team - this will be documented
+per repo.
+
+### Team member rotation
+
+Initially rotations will be hand selected. We are working on integrating with tools
+to make the current on-call more discoverable and alerts to be sent automatically to
+them.
+
+## Communication norms
+
+When responding to an incident, communicate early and often:
+
+1. Acknowledge ("ack") an issue where it is reported and announce the actions
+   you are taking or delegating clearly. Do this in the corresponding channel
+   you will be provided access to.
+2. When an issue or incident affects multiple developers or will take time to
+   resolve, consider announcing that to affected developers and filing a
+   tracking issue on GitHub.
+
+Primary communication about the changes in a commit should happen on the pull
+request that commit came from. This ensures that members of the open source
+community (who may not be members of Discord or Slack servers) can see the
+discussion and avoid duplicating effort.
```

Comment:
Could replace "Discord or Slack servers" with "other communication channels" (to cover Teams, email threads, etc. too)